package.json

```diff
@@ -8,6 +8,8 @@
     "@chakra-ui/react": "^2.8.0",
     "@emotion/react": "^11.11.1",
     "@emotion/styled": "^11.11.0",
+    "@fontsource/open-sans": "^5.0.7",
+    "@fontsource/raleway": "^5.0.7",
```

Comment:
NON-BLOCKING: Are we actually using these fonts?

Reply:
I don't see them being used anywhere and the new fonts show up without me installing these.